### PR TITLE
feat: create flow - hide nft collection configuration when no nft added [1/n]

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

Hides NFT configuration settings on NFT Page in create when there is no NFTs added yet.

I didn't add the skip button as its a pretty big hassle. Unsure the ROI is worth it.

Also added a Goverance type review to r&deploy

Closes https://github.com/jbx-protocol/juice-interface/issues/2901

## Screenshots or screen recordings

image

image

image

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3015** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3015" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3015?utm_source=stack-comment).

File: src/components/Create/Create.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { t, Trans } from '@lingui/macro'
-import { Callout } from 'components/Callout'
 import { DeployButtonText } from 'components/buttons/DeployProjectButtonText'
+import { Callout } from 'components/Callout'
```
Comment (wraeth-eth) [inline src/components/Create/Create.tsx:3]: Change isn't actually anything, just a format on save accident

File: src/components/Create/components/pages/NftRewards/NftRewardsPage.tsx [hunk 2]
```diff
@@ -57,210 +61,218 @@ export const NftRewardsPage = () => {
...
-              }
-              required
-            >
-              <JuiceInput />
-            </Form.Item>
-
-            <Form.Item
-              name="collectionDescription"
-              label={<Trans>Collection Description</Trans>}
+          {hasNfts && (
```
Comment (wraeth-eth) [inline src/components/Create/components/pages/NftRewards/NftRewardsPage.tsx:64]: None of this code changed, just nested due to this line here to toggle hidden or not

File: src/components/Create/components/pages/ReviewDeploy/components/RewardsReview/RewardsReview.tsx [hunk 3]
```diff
@@ -81,6 +94,13 @@ export const RewardsReview = () => {
             </div>
           }
         />
+        <DescriptionCol
```
Comment (wraeth-eth) [inline src/components/Create/components/pages/ReviewDeploy/components/RewardsReview/RewardsReview.tsx:97]: We weren't showing onchain goverance in NFT settings, so quickly added. dont @ me

**Review by tomquirk — APPROVED** [review]
love @wraeth-eth lgtm

**Comment by wraeth-eth** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Job ID: srBI9sfe6ZNRXrauhrGk.
See details on [graphite.dev](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/3015)

**Pull request merged**